src/test/java/de/tum/in/www1/artemis/bonus/BonusIntegrationTest.java

```diff
@@ -295,7 +298,15 @@ void testCalculateRawBonusWithGradesContinuousBonusStrategy() throws Exception {
         bonusToExamGradingScale = gradingScaleRepository.findWithEagerBonusFromByExamId(bonusToExam.getId()).orElseThrow();
         gradingScaleRepository.deleteAll(List.of(bonusToExamGradingScale, courseGradingScale));
 
-        GradingScale sourceGradingScale = createSourceGradingScaleWithGradeStepsForGradesBonusStrategy(sourceCourse);
+        GradingScale sourceGradingScale;
+        if (isContinuous) {
+            sourceGradingScale = createSourceGradingScaleWithGradeStepsForGradesBonusStrategy(sourceCourse);
+
```

Comment:
maybe remove this empty line, not really needed

Reply:
removed it :+1: